util/ubuntu-setup

```diff
@@ -65,6 +71,31 @@ if [ $UID -ne 0 ]; then
 	exec sudo $0 $@
 fi
 
+if $OVERWRITE_SOURCES; then
+    read -p "Do you really want to overwrite your sources.list? THIS SHOULD ONLY BE DONE ON UBUNTU 14.04 [yN]: " yn
+    case $yn in
+        [Yy]* ) ;;
+        [Nn]* ) exit;;
+        * ) exit ;;
+    esac
+
+    # Copy sources.list to sources.list.old if sources.list.old is empty
+    if [ -f "/etc/apt/sources.list" ]; then
```

Comment:
This is correct, I think. 
```
If sources.list is found
rename sources.list -> sources.list.old (with no overwrite)
```

If sources.list does not exist, then we should not move it (that would fail). Maybe we should have it error out, as that usually means you are on a non-debian system.